test/cypress/support/editMode.ts

```diff
@@ -64,6 +64,12 @@ const commands = {
     setSearchableDropdownConfigurationField: (widgetId: string, fieldName: string, value: string) =>
         cy.editWidgetConfiguration(widgetId, () => {
             cy.setSearchableDropdownValue(fieldName, value);
+        }),
+
+    setDropdownConfigurationField: (widgetId: string, fieldName: string, value: string[]) =>
```

Comment:
1. I think more accurate would be to have 
```suggestion
    setDropdownConfigurationField: (widgetId: string, fieldName: string, values: string[]) =>
```
2. You use it for multiple dropdown, but it will work also for single selection dropdowns, right?
3. Maybe `clearDropdown` should be done inside `setDropdownValues`?

Reply:
1. fixed 
2. should work
3. no, imho having it separately, keep current implementation of `set Dropdown Values` does not change current behavior of this method